### ISSUE TITLE
Can't install h2o_pysparkling_2.0 on HDInsight Spark cluster

### ISSUE THREAD

**Issue by jakehockey10** [h2oai/sparkling-water#229]
In Azure, I have an init script action that runs on the nodes with the following content:

``` bash
#!/bin/sh

# Anything with —target specified is where the notebook sources its packages from

# install pip
wget https://bootstrap.pypa.io/get-pip.py -O install_pip.py
sudo -H python install_pip.py
rm install_pip.py

sudo apt-get -y install python-requests

# install h2o
sudo -H pip install --target=/usr/bin/anaconda/lib/python2.7/site-packages h2o
sudo -H pip install --target=/usr/hdp/current/spark2-client/python h2o

# install pysparkling
sudo -H pip install --target=/usr/bin/anaconda/lib/python2.7/site-packages h2o_pysparkling_2.0
sudo -H pip install --target=/usr/hdp/current/spark2-client/python h2o_pysparkling_2.0

# install h5spark dependencies
sudo -H pip install --target=/usr/bin/anaconda/lib/python2.7/site-packages h5py
sudo -H pip install --target=/usr/hdp/current/spark2-client/python h5py

# download h5spark and put in proper directory
wget https://github.com/valiantljk/h5spark/archive/master.zip
unzip master.zip -d h5spark
rm master.zip
sudo cp -R /h5spark/h5spark-master/src/main/python/h5spark usr/hdp/current/spark2-client/python
rm -rf /h5spark

# Mount azure fileshare on worker nodes
# stuff ommitted
exit 0
```

Just a little bit of context: I'm successfully pulling HDF5 files from blob storage into a Jupyter notebook and converting a dataset in that file into an RDD.  I'd like to create an H2OFrame from the RDD, which requires `pysparkling`.

I can't create an `H2OContext`, and so I assumed that it was because the installation of `pysparkling` did not succeed.  Sure enough, if I look at the tasks output of that script action, I see the following error:

```
...

Collecting tabulate (from h2o)
  Using cached tabulate-0.7.7-py2.py3-none-any.whl
Installing collected packages: requests, colorama, future, tabulate, h2o
Successfully installed colorama-0.3.7 future-0.16.0 h2o-3.10.4.2 requests-2.13.0 tabulate-0.7.7
Collecting h2o_pysparkling_2.0
  Downloading h2o_pysparkling_2.0-2.0.6.tar.gz
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-build-Po14Su/h2o-pysparkling-2.0/setup.py", line 13, in <module>
        with open(path.join(here, 'build', 'version.txt'), encoding='utf-8') as f:
      File "/usr/lib/python2.7/codecs.py", line 896, in open
        file = __builtin__.open(filename, mode, buffering)
    IOError: [Errno 2] No such file or directory: '/tmp/pip-build-Po14Su/h2o-pysparkling-2.0/build/version.txt'
    
    ----------------------------------------
Collecting h2o_pysparkling_2.0
  Using cached h2o_pysparkling_2.0-2.0.6.tar.gz
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-build-XsU1SD/h2o-pysparkling-2.0/setup.py", line 13, in <module>
        with open(path.join(here, 'build', 'version.txt'), encoding='utf-8') as f:
      File "/usr/lib/python2.7/codecs.py", line 896, in open
        file = __builtin__.open(filename, mode, buffering)
    IOError: [Errno 2] No such file or directory: '/tmp/pip-build-XsU1SD/h2o-pysparkling-2.0/build/version.txt'
    
    ----------------------------------------
Collecting h5py
  Using cached h5py-2.7.0-cp27-cp27mu-manylinux1_x86_64.whl
Collecting numpy>=1.7 (from h5py)
  Using cached numpy-1.12.1-cp27-cp27mu-manylinux1_x86_64.whl

...
```

Does anyone have an idea of what I'm doing wrong?

**Comment by jakubhava** [comment]
Hi @jakehockey10,
 we already have a fix prepare and will go into the next sparkling-water release - 2.1.5

**Comment by jakubhava** [comment]
Hi @jakehockey10 please use this JIRA to track the issue https://0xdata.atlassian.net/browse/SW-335. The fix is planned to go into 2.1.5 release

**Comment by jakehockey10** [comment]
@jakubhava, thanks for the update!
